main/test/fixtures/server.js

```diff
@@ -243,6 +242,7 @@ export async function createServer() {
                 }
                 await getReaderValue(session.incomingUnidirectionalStreams)
                 await session.close()
+                // eslint-disable-next-line no-unused-vars
               } catch (error) {
                 // do not crash server, if a problem occurs...
               }
```

Comment:
If you want to ignore the error you can just omit it, then there's no need to disable linting:

```js
} catch {
  // do not crash server, if a problem occurs...
}
```